humble/distribution.yaml

```
@@ -3803,6 +3803,16 @@ repositories:
       url: https://github.com/ros-visualization/interactive_markers.git
       version: humble
     status: maintained
+  inverse_dynamics_solver:
+    doc:
+      type: git
+      url: https://github.com/unisa-acg/inverse-dynamics-solver.git
+      version: v0.1.0
+    source:
+      type: git
+      url: https://github.com/unisa-acg/inverse-dynamics-solver.git
+      version: v0.1.0
```

Please put a branch name instead of tag name here. It should be the branch on which you develop changes for ROS Humble.

```
ERROR: In file '''/home/runner/work/rosdistro/rosdistro/humble/distribution.yaml''': Could not validate source entry for repo inverse_dynamics_solver with error [[[Could not validate repository with url https://github.com/unisa-acg/inverse-dynamics-solver.git and version v0.1.0 from entry at line 3811. Error reason: Tags are not valid, but a tag v0.1.0 was found. Re: https://github.com/ros/rosdistro/pull/20286]]]
```

Often it's the name of the ROS distro itself (ex: `version: humble`). In this case, it looks like your repo has only one branch named `main`.

```suggestion
      version: main
```

If you decide you want to support multiple ROS distributions, I recommend having a separate branch for each one. That way you can choose what changes to backport to Humble, and avoid making API or ABI breaking changes.

---

@sloretz thank you for your comment. I have updated the PR accordingly.